dropwizard/src/main/resources/META-INF/services/io.dropwizard.jackson.Discoverable

```diff
@@ -1 +0,0 @@
-com.linecorp.armeria.dropwizard.logging.AccessLogWriterFactory
```

Comment:
Is this (and related files) really not needed?

Or did you just remove the `@JsonTypeInfo` usages?

Reply:
I think we can remove it. 
Because:
1) The `AccessLogWriter` type and format can be configured via `AccessLog` in `ArmeriaSettings`.
https://github.com/line/armeria/blob/3d6d80565550183f2d317a3068b48ad6338b6ab2/dropwizard/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaSettings.java#L919-L955
2) The `AccessLogWriter` is set to `ServerBuilder` statically in `configureAccessLog`
https://github.com/line/armeria/blob/7413d196d7920dbfd9839d6017826caa01180713/dropwizard/src/main/java/com/linecorp/armeria/dropwizard/ArmeriaConfigurationUtil.java#L469-L478

Let me know if there is something I missed.

Reply:
Just making sure it wasn't needed anymore. 

Ref. https://www.dropwizard.io/en/stable/manual/configuration.html#polymorphic-configuration

Reply:
Personally, I think statically assigned if-else (or switch on string) are code-smells. 

https://refactoring.guru/replace-conditional-with-polymorphism